Web/Command_Injection.md

```diff
@@ -0,0 +1,35 @@
+/*
+Title: Command Injection
+Description: Injection of commands through web application onto host machine.
+*/
+
+- LAST UPDATED DATE: 2015/12/1
+- LAST UPDATED BY: @zaeyx
+
+## Summary
+
+Command Injection occurs when an insecure application passes unsafe user supplied content to a system shell.  
+
+## Capabilities and Risk
+
+When Command Injection occurs, and attacker may be able to execute arbitrary commands as the web application's host machine.  This gives effective control over whatever portion of the host machine the web server's user is given access to.  
+
+## Detection
+
+Detection can be accomplished by searching for command line access by the web server's user that is not expected to have been given by the normal operations of the application.  For example, if an application uses command line operations to perform a ping.  If the web server's user is executing any command other than "ping" you might have a problem.
```

Comment:
How would you suggest people "search" for Command Injection? Are there any tools that would allow you to detect this? Can you give examples on Windows and Linux?